packages/runtime/src/useCases/transport/identity/CheckIdentity.ts

```diff
@@ -47,25 +47,30 @@ export class CheckIdentityUseCase extends UseCase<CheckIdentityRequest, CheckIde
         const relationship = await this.relationshipsController.getRelationshipToIdentity(address);
         if (relationship) {
             const relationshipDTO = RelationshipMapper.toRelationshipDTO(relationship);
-            if (relationship.status === RelationshipStatus.Pending) {
-                return Result.ok({
-                    peer: true,
-                    relationshipPending: true,
-                    relationship: relationshipDTO
-                });
-            } else if (relationship.status === RelationshipStatus.Active || relationship.status === RelationshipStatus.Terminating) {
-                return Result.ok({
-                    peer: true,
-                    relationshipActive: true,
-                    relationship: relationshipDTO
-                });
+            switch (relationship.status) {
+                case RelationshipStatus.Pending:
+                    return Result.ok({
+                        peer: true,
+                        relationshipPending: true,
+                        relationship: relationshipDTO
+                    });
+                case RelationshipStatus.Active:
+                    return Result.ok({
+                        peer: true,
+                        relationshipActive: true,
+                        relationship: relationshipDTO
+                    });
+                case RelationshipStatus.Terminated:
+                    return Result.ok({
+                        peer: true,
+                        relationshipTerminated: true,
+                        relationship: relationshipDTO
+                    });
+                default:
+                    return Result.ok({
+                        unknown: true
+                    });
```

Comment:
```suggestion
                    return Result.ok({
                        peer: true,
                        relationship: relationshipDTO
                    });
```

Reply:
I mean ... it found a relationship so it is definitely not unknown.

Reply:
what is the peer vs. unknown distinction used for? The remaining cases are rejected and revoked

Reply:
unknown means you have never heard of defined the address, if you find a relationship you defenitely have contacted the relationship before.